src/averageAges.js

```diff
@@ -20,6 +20,14 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = people.filter(person => person.sex === 'm');
+  const filteredMen = (century
+    ? men.filter(man => Math.ceil(man.died / 100) === century)
+    : men);
+  const averageMenAges = filteredMen.reduce((sum, man) => (
```

Comment:
If you want, you can use function for calculate average and use it in 27 and 52 lines.